test/cppsuite/tests/run.cpp

```diff
@@ -154,8 +157,8 @@ main(int argc, char *argv[])
 {
     std::string cfg, config_filename, current_cfg, current_test_name, test_name, wt_open_config;
     int64_t error_code = 0;
-    const std::vector<std::string> all_tests = {"burst_inserts", "hs_cleanup", "operations_test",
-      "search_near_01", "search_near_02", "search_near_03", "test_template"};
+    const std::vector<std::string> all_tests = {"cache_resize", "burst_inserts", "hs_cleanup",
```

Comment:
Minor: consider alphabetising

Reply:
Done in 61ec91a6c7e81cca5a5a26447df5ef93fd134b97